Zajecia/src/main/java/pl/isa/javasmugglers/UserLogin.java

```diff
@@ -0,0 +1,89 @@
+package pl.isa.javasmugglers;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+import java.io.IOException;
+import java.util.*;
+
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonMappingException;
+
+public class UserLogin {
```

Comment:
To trzeba podzielić na 2 klasy.
1. Jest widokiem, który wyświetla informacje użytkownikowi i wczytuje od niego dane.
2. Jest kontrolerem, który zwraca użytkowników lub konkretnego użytkownika, pozwala na dodawanie i wykonuje operacja na plikach JSON.

Reply:
A tu mógłbym to rozwiązać przez klasy wewnętrzne? Tak jak w commit 6? albo jakaś mała wskazówka jak to dobrze rozdzierlić, bo powiem szczerze, że nie mam pomysłu.